### ISSUE TITLE
Review "borrowed" graphical elements

### ISSUE THREAD

**Issue by razcore-art** [GDQuest/website#119]
For example the 3 main logos of the software we use are inconsistent. Krita's is all fuzzy and I just checked their logo - the one they use isn't. So it's either a scale-up issue or an older logo perhaps.

![image](https://user-images.githubusercontent.com/1177508/70349787-1c63e780-186e-11ea-87f1-ff1e15f08629.png)

Anyway, we should go over all graphical elements, including screenshots on tutorial/guidelines etc. pages and try to keep quality consistent.

**Comment by NathanLovato** [comment]
Logos from software or other projects are their logos, we're not going to change them. But we should certainly update it yes. The entire landing page needs a remake anyway, these cards will certainly go away - they don't have a place there anymore. See #106

**Comment by razcore-art** [comment]
This is where you took "the liberty" of interpreting what I said without me even thinking it :P. I never said or was not my intention to imply that we should modify the logos to keep them consistent. That's why I said I checked Krita's logo specifically. Their actual logo isn't fuzzy, it's actually vector and the paint stroke isn't painterly either, it's a shape. So this is probably an older version of the logo. It looks bad anyway, especially when put near the other crispy versions.

The idea was/is that even if it looks bad cause it's a different style, we could probably come up with a solution place it another way, have them scaled in a way to make them unite more etc.

I know some of the stuff is going to go away, that's exactly why I opened the issue. When it goes away, some things need to be revised and unified - including screenshot quality and what not.
